tests/UnitTests/CoreGraphics.drawing/DrawingTest.cpp

```diff
@@ -62,13 +62,11 @@ CFStringRef testing::DrawTest<TComparator>::CreateAdditionalTestDescription() {
 
 template <typename TComparator>
 CFStringRef testing::DrawTest<TComparator>::CreateOutputFilename() {
-    const ::testing::TestInfo* const test_info = ::testing::UnitTest::GetInstance()->current_test_info();
     woc::unique_cf<CFStringRef> additionalDesc{ CreateAdditionalTestDescription() };
     woc::unique_cf<CFStringRef> filename{ CFStringCreateWithFormat(nullptr,
                                                                    nullptr,
-                                                                   CFSTR("TestImage.%s.%s%s%@.png"),
-                                                                   test_info->test_case_name(),
-                                                                   test_info->name(),
+                                                                   CFSTR("TestImage.%s%s%@.png"),
+                                                                   GET_TEST_FULL_NAME().c_str(),
```

Comment:
I haven't looked through how we're using TAEF logging, but Log::File(L"path to file") would result in the image being saved under TAEF's current output directory.  

From the TAEF docs;

Log:: File(const wchar_t* pszFileName) 
Log a test file to be saved. Files are saved to a folder named "WexLogFileOutput". This folder will be in the folder specified by the WTTRunWorkingDir environment variable if that variable exists. When using Wex.Logger outside of TAEF and the WTTRunWorkingDir environment variable does not exist, the WexLogFileOutput folder will be in the current directory. When using Wex.Logger along with TAEF and the WTTRunWorkingDir environment variable does not exist, the WexLogFileOutput folder will be in the TAEF output folder.

If you just want to log the path to the file, you could Log::Property(const wchar_t* pszName, const wchar_t* pszValue).  Of course, we'll need wttlog.dll to generate test logs.

From the TAEF docs;

Log::Property(const wchar_t* pszName, const wchar_t* pszValue) 
Log a name/value property pair. The value can be in xml format.

---
In reply to: [98139951](https://github.com/Microsoft/WinObjC/pull/1849#discussion_r98139951) [](ancestors = 98139951)